orchagent/muxorch.cpp

```diff
@@ -1208,9 +1213,37 @@ bool MuxOrch::handleMuxCfg(const Request& request)
     auto srv_ip = request.getAttrIpPrefix("server_ipv4");
     auto srv_ip6 = request.getAttrIpPrefix("server_ipv6");
 
+    std::set<IpAddress> skip_neighbors;
+    std::string cable_type = "active-standby";
```

Comment:
We don't have to check for cable_type in orchagent. If 'soc_ipv4' is present, it must be skipped from tunnel neighbors. Please remove.